src/pages/data-entry/data-entry.jsx

```diff
@@ -132,64 +219,66 @@ export class DataEntryForm extends Component {
             <br />
             <br />
           </span>
-          <label htmlFor="Entry">
+          <label htmlFor='Entry'>
             Enter Height and it&apos;s Units:
             <input
-              className="form-control"
-              type="input"
-              name="height"
-              placeholder="Height is..."
+              className='form-control'
+              type='input'
+              name='height'
+              placeholder='Height is...'
               onChange={this.addheight}
             />
             <input
-              className="form-control"
-              type="input"
-              name="hunits"
-              placeholder="in, ft, cm, or m"
+              className='form-control'
+              type='input'
+              name='hunits'
+              placeholder='in, ft, cm, or m'
               onChange={this.addheightUnits}
             />
             <React.Fragment>
               {this.state.errorhunits ? (
-                <h1>Invalid entry for units of height</h1>
+                <h1>
+                  We do not recognize this unit, and will ask you about it.
+                </h1>
               ) : null}
             </React.Fragment>
           </label>
           <span key={2}>
             <br />
             <br />
           </span>
-          <label htmlFor="Entry">
-            {" "}
-            
-          Enter Weight and it&apos;s Units:
-           
+          <label htmlFor='Entry'>
+            {' '}
+            Enter Weight and it&apos;s Units:
             <input
-              className="form-control"
-              type="input"
-              name="weight"
-              placeholder="Weight is..."
+              className='form-control'
+              type='input'
+              name='weight'
+              placeholder='Weight is...'
               onChange={this.addweight}
             />
             <input
-              className="form-control"
-              type="input"
-              name="wunits"
-              placeholder="lb or kg"
+              className='form-control'
+              type='input'
+              name='wunits'
+              placeholder='lb or kg'

```

Comment:
Do lb, open to discuss.

Reply:
Should we not offer them either though? the whole point of this is we accept multiple types of units, so why not initialize with some options we know about?

Reply:
Okay, you are right. Reason I say that is because our charts are based on lb right now. Can we make the message steer more towards lb. "lb (other units acceptable)", something to that extent.